web/client/components/misc/coordinateeditors/editors/DecimalCoordinateEditorSearch.jsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2024, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+*/
+
+import {capitalize} from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {FormGroup} from 'react-bootstrap';
+import DecimalCoordinateEditor from './DecimalCoordinateEditor';
+import IntlNumberFormControl from '../../../I18N/IntlNumberFormControl';
+
+/**
+ This component renders a custom coordiante inpout for decimal degrees for default coordinate CRS and current map CRS as well
+*/
+class DecimalCoordinateEditorSearch extends DecimalCoordinateEditor {
```

Comment:
The component name can be distinctive. Modify it in other places too.
Also check if extending DecimalCoordinateEditor is still needed.

```suggestion
class CRSCoordinateEditor extends DecimalCoordinateEditor {
```